docs/index.html

```diff
@@ -8,6 +8,11 @@
   <meta name="description" content="Description">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
   <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/lib/themes/vue.css">
+  <style>
+    html {
+      --theme-color:#c4a604;
```

Comment:
This has a contrast ratio of 2.38, normal text (which this resource would be used for) should have a ratio of 4.5. @yaichenbaum Is this the "theme" color to use and if so what do you suggest to do with this?

Reply:
@chingucoding I was thinking we should use something closer to `#E89609`.

Reply:
Hmm I see. The contrast ratio isn't ideal too @yaichenbaum:

![Screenshot showing bright orange text on white background](https://user-images.githubusercontent.com/16122379/110009674-a030ac00-7d1d-11eb-8f6e-fdc2093a047b.png)

Not sure if it's a sacrifice worth making but given that links will use this color, having good contrast ratios should be a goal.

Reply:
@chingucoding That was just a general idea for which direction to move in, do you have any suggestions for a color with better contrast ratios?

Reply:
I see. For 3:1 ratio you could use `#b37407`, for 4.5:1 you could use `#a66907` or `#a36507`. Unfortunately, dark orange is just brown...

Reply:
Let's try `#a66907`, and if we want, we can change it later.

Reply:
```suggestion
      --theme-color:#a66907;
```

To make it easier for you @pklion

Reply:
Oh, I didn't think about the contrast ratio.
Thank you!